.travis.yml

```diff
@@ -56,6 +56,7 @@ script:
   - docker exec jcsda_container ls
   - docker exec jcsda_container ls /jcsda/src_repo
   - docker exec jcsda_container bash -c 'cp -r /jcsda/.openmpi/ ~/'
+  - docker exec jcsda_container bash -c 'pip install xarray' 
```

Comment:
Needed for now but we need to remember to remove this when we move to the new containers